app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -744,6 +753,14 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
         }
     }
 
+    open fun navigateToBrowserOnColdStart() {
```

Comment:
Let's cover this one with a unit test. Gave it a try to see how painful it is. I think we might want to refactor `HomeActivityTest` later, but for now something like this will do: 

```kotlin
@Test
    fun `navigateToBrowserOnColdStart`() {
        clearMocks(activity)
        
        val browsingModeManager: BrowsingModeManager = mockk()
        every { browsingModeManager.mode } returns BrowsingMode.Normal

        val settings: Settings = mockk()
        every { settings.shouldReturnToBrowser } returns true
        every { activity.components.settings.shouldReturnToBrowser } returns true
        every { activity.openToBrowser(any(), any()) } returns Unit

        activity.browsingModeManager = browsingModeManager
        activity.navigateToBrowserOnColdStart()

        verify(exactly = 1) { activity.openToBrowser(BrowserDirection.FromGlobal, null) }
    }
```

(We can also over the case where `browserModeManager.mode.isPrivate` is `true`)